packages/core/echo/echo-schema/src/index.ts

```diff
@@ -6,7 +6,7 @@ export * from './database';
 export * from './database-router';
 export * from './defs';
 export * from './object';
-export * from './ordered-set';
+export * from './echo-array';
```

Comment:
please sort import/exports

Reply:
isn't the linter doing it?

Reply:
can you try to run lint before submitting your PR?